Cargo.lock

```diff
@@ -2935,7 +2966,7 @@ version = "0.18.0-alpha.1+dev"
 dependencies = [
  "anyhow",
  "clap",
- "glam",
+ "glam 0.22.0",
```

Comment:
This is annoying - I think it's high time to tackle https://github.com/rerun-io/rerun/issues/6467

I'll get on it

Reply:
> Forking macaw I noticed we have a `MeshGen` in there, which we could probably use instead of `hexasphere`,

I see! I also see that it produces "lat-long" spheres. This is potentially good, if the data is polar in some way, or for future extensions to capsules, but also bad, because that type of sphere generation has an uneven density of lines — dense near the poles and sparse near the equator.

It might make sense to offer both. I can certainly drop `hexasphere` in this iteration of the PR, but I think for the first version it makes sense to offer a more regular, non-axis-privileging mesh structure rendering. What do you think?

Reply:
I agree that the hexaspheres probably look better. I think we can stick with those for now. We want to merge https://github.com/rerun-io/rerun/pull/6867 first though to avoid duplicated glam dependencies